package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack"
```

Comment:
- [ ] kindly include a command to run your webpack in dev mode e.g `"serve": "webpack serve --open "`